hotxlfp/formulas/utils.py

```diff
@@ -25,7 +25,7 @@
     "=": operator.eq,
     ">=": operator.ge,
     "<=": operator.le,
-    "^": lambda a, b: torch.pow(torch.tensor(a), torch.tensor(b)),
+    "^": lambda a, b: torch.float_power(torch.tensor(a), torch.tensor(b)),
```

Comment:
Fixes another bug where power with negatives didn't work. For example

`2^(-1)` would be 0